coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/PropDevLookupableHelperServiceImpl.java

```diff
@@ -304,6 +306,11 @@ public void buildPropDevEditActionLink(Link actionLink, Object model,String titl
 
 	}
 
+    protected boolean isInWorkflow(String docId) throws WorkflowException{
+        String documentStatus = getWorkflowDocumentService().getDocumentStatus(docId).getCode();
```

Comment:
Couldn't we pass in the PD doc which has the DocumentHeader where you can get the status.